tests/no-explicit-type-exports.test.ts

```diff
@@ -88,43 +88,70 @@ ruleTester.run('no-explicit-type-exports', rule, {
       filename: fileName,
       code: "import type * as types from './bar'; export {types};",
     },
+    {
```

Comment:
Tests for locally defined interfaces and types, an explicit check that a function const will pass, and that a single line import/export passes with a function const named similarly.